core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -77,7 +84,7 @@ public static void main(String[] args) {
       System.exit(-1);
     }
     try {
-      format(args[0]);
+      format(Mode.valueOf(args[0]));
```

Comment:
we should convert args to uppercase, and double check that the error message is helpful when args[0] is something other than "master" or "worker"